Composer/packages/client/src/pages/setting/extensions/InstallExtensionDialog.tsx

```diff
@@ -0,0 +1,165 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/** @jsx jsx */
+import { jsx } from '@emotion/core';
+import React, { useState, useEffect } from 'react';
+import { Dialog, DialogType, DialogFooter } from 'office-ui-fabric-react/lib/Dialog';
+import { TextField } from 'office-ui-fabric-react/lib/TextField';
+import { DefaultButton, PrimaryButton } from 'office-ui-fabric-react/lib/Button';
+import {
+  DetailsListLayoutMode,
+  SelectionMode,
+  IColumn,
+  CheckboxVisibility,
+} from 'office-ui-fabric-react/lib/DetailsList';
+import { ShimmeredDetailsList } from 'office-ui-fabric-react/lib/ShimmeredDetailsList';
+import axios from 'axios';
+import formatMessage from 'format-message';
+
+import httpClient from '../../../utils/httpUtil';
+
+// TODO: extract to shared?
+type ExtensionSearchResult = {
```

Comment:
this is being used in `@bfc/extension` as well, so might as well move it to shared

Reply:
My plan is to create an extensions shared package eventually. So I'm fine with some duplication in the meantime.